includes/shallowbackup.js

```diff
@@ -14,61 +14,78 @@
 'use strict';
 
 const async = require('async');
+const error = require('./error.js');
+const events = require('events');
 const request = require('./request.js');
 
-module.exports = function(dbUrl, blocksize, parallelism, log, resume) {
+module.exports = function(dbUrl, limit, parallelism, log, resume) {
```

Comment:
I think we should have a comment here regarding the signature parameters that are unused.